### PR TITLE
PLATFORM-3302 wgUploadPath https

### PR DESCRIPTION
**Pull request by harnash**
This script is currently running in dry-run mode. After it's finished I'll verify and run it again without dry-run.

ping: @Wikia/core-platform-team

### CODE REVIEW

File: maintenance/wikia/WikiFactoryVariables/migrateWikiFactoryToHttps.php [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+	protected $dryRun  = false;
+	protected $verbose = false;
+	protected $varName = '';
+
+	public function __construct() {
+		parent::__construct();
+		$this->mDescription = "Migrates Wiki Factory variable to HTTPS";
+		$this->addOption( 'dry-run', 'Dry run mode', false, false, 'd' );
+		$this->addOption( 'varName', 'Name of the Wiki Factory variable', true, true, 'k' );
```
Comment (themech) [inline maintenance/wikia/WikiFactoryVariables/migrateWikiFactoryToHttps.php]: nit: either dashes or lowerCamelCase, `dry-run` and  `varName` next to each other look bad

File: maintenance/wikia/WikiFactoryVariables/migrateWikiFactoryToHttps.php [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+				return false;
+			}
+		}
+
+		$oldValue = $keyValue = $varData;
+
+		if ( empty( $keyValue ) ) {
+			$this->output( "Variable is empty for $wgCityId - skipping" . PHP_EOL );
+
+			if ( $fh ) {
```
Comment (themech) [inline maintenance/wikia/WikiFactoryVariables/migrateWikiFactoryToHttps.php]: This `if`/`fclose` pair is repeated too many times, could you please move it to `finally` section?

**Review by themech — CHANGES_REQUESTED** [review]

File: maintenance/wikia/WikiFactoryVariables/migrateWikiFactoryToHttps.php (hunk 1 above)
Comment (harnash) [reply maintenance/wikia/WikiFactoryVariables/migrateWikiFactoryToHttps.php]: fix: 902f839ebe

File: maintenance/wikia/WikiFactoryVariables/migrateWikiFactoryToHttps.php (hunk 2 above)
Comment (harnash) [reply maintenance/wikia/WikiFactoryVariables/migrateWikiFactoryToHttps.php]: I did it in a bit different way: 3a984026cf8

**Review by themech — APPROVED** [review]

**Pull request merged**